Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -485,11 +485,26 @@ section NormedRing
 
 variable [NormedRing α]
 
-theorem Units.norm_pos [Nontrivial α] (x : αˣ) : 0 < ‖(x : α)‖ :=
-  norm_pos_iff.mpr (Units.ne_zero x)
+namespace Units
 
-theorem Units.nnnorm_pos [Nontrivial α] (x : αˣ) : 0 < ‖(x : α)‖₊ :=
-  x.norm_pos
+theorem norm_pos [Nontrivial α] (x : αˣ) : 0 < ‖(x : α)‖ := norm_pos_iff.mpr (Units.ne_zero x)
+
+theorem nnnorm_pos [Nontrivial α] (x : αˣ) : 0 < ‖(x : α)‖₊ := x.norm_pos
+
+lemma norm_commutator_sub_one_le {a b : αˣ} :
```

Comment:
Is this true in a seminormed ring?

Reply:
It's actually stated like that in my repo. Looks like I pasted in the wrong section